wiremock/src/main/java/com/cognifide/aem/stubs/wiremock/WireMockStubs.java

```diff
@@ -193,5 +206,17 @@ public WireMockServlet getServlet() {
         + " for response body content and file paths when loading body files. Effectively enables dynamic file loading"
         + " instead of preloading and simplifies defining stubs.")
     boolean globalTransformer() default true;
+
+    @AttributeDefinition(
+      name = "Request journal",
+      description = "Enable the request journal, which records incoming requests for later verification. For long running instances can exhaust the heap."
```

Comment:
For long-running instances can exhaust the heap."

\=> Protects against reserving too much memory.  
  
how about enabling if by default with sensitive max? if exceeding max will behave reasonably

Reply:
I would keep it disabled as we never know how it could be used and in what setup.